decidim-initiatives/config/locales/en.yml

```diff
@@ -66,13 +66,10 @@ en:
         search_placeholder:
           title_or_description_cont: Search %{collection} by title or description.
         state_eq:
-          label: State
           values:
-            accepted: Accepted
             created: Created
             discarded: Discarded
             published: Published
-            rejected: Rejected
```

Comment:
If this strings have been reused, doesn't it mean that they were already being used before? If we remove them won't it affect the initiatives module?

Reply:
the initiatives modules will take the string from the proposals module, this string is moved there.
The problem now is that the proposals module is using this string and, if initiatives is not active, doesn't find it.
If we keep it here, it will be repeated in 2 files isn't?

Reply:
I get the problem. But I'm referring to the remote (but possible?) case where initiatives are enabled and proposals not. Also, to have the code correctly grouped, each module should be as independent as possible, and it seems to me that this should not be a reason to have this dependency.

Reply:
In fact I agree with you. The thing is that is that many strings are created this way, this means that we will should "namespace" all strings in all modules. That's not a quick fix. We should create a issue for that and treat it properly.
I any case, I don't thing it is possible to run decidim without the proposals module.

Reply:
>  I any case, I don't thing it is possible to run decidim without the proposals module.

Yes, that's the case right now, it's one of the (too many) modules that we've as a dependency by default. The thing is that shouldn't be, as we're starting to see [some instances](https://petitions.senat.fr/initiatives) that have only Initiatives module enabled. 

But I also agree on having this discussion on another place, and that for the moment this PR is OK (proposals can be an implicit depedency of initatives; there are cases where initiatives are not enabled)